app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -51,6 +51,18 @@ public function __construct()
         ]]);
     }
 
+    private function isDoublePost($post)
```

Comment:
I didn't add this block to the Topic model because it contains a `Auth::` check which doesn't really fit in there. Passing `user_id` might work but seems unnecessary. But anyway, I moved it.